README.md

```diff
@@ -48,6 +48,16 @@ npm run-script build
 Then include `dist/web3.js` in your html file.
 This will expose `Web3` on the window object.
 
+Or via jsDelivr CDN
+
+```html
+<script src="https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.x/dist/web3.min.js"></script>
```

Comment:
```suggestion
<script src="https://cdn.jsdelivr.net/npm/web3@latest/dist/web3.min.js"></script>
```

Thanks for proposing this PR! We will merge this as soon as we released a new version on npm with the updated minified file from PR https://github.com/ethereum/web3.js/pull/3261.